frontend/app/controllers/sign-in.js

```diff
@@ -0,0 +1,68 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+/*---- Login Screen ---*/
+
+  loginFailed: false,
+  loginFailedMessage: "Failed to Login",
+  isProcessing: false,
+  isSlowConnection: false,
+  timeout: null,
+
+    success: function(response) {
```

Comment:
@niranjan94 Terrific. This means I can start making progress now.
Thank you for your time. I'll make the changes, and bother you again if necessary